src/scss/_badges.scss

```diff
@@ -20,6 +20,21 @@
 	color: $text-primary;
 }
 
+.label-default {
+	background-color: $label-default-bg;
```

Comment:
@fuhton Now that you've overwritten the variable, Bootstrap will have this style already. This can be removed.